src/scenes/Login/Login.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react'
+import './_login.scss'
+import fetch from 'node-fetch'
+import PropTypes from 'prop-types'
+import Home from '../Home/Home'
+const Login = ({ setFoundUser }) => {
+  const [username, setUsername] = useState()
+  const [password, setPassword] = useState()
+  const [error, setError] = useState(null)
+  const [loading, setLoading] = useState(false)
+  const loginUser = async () => {
+    return fetch(`http://localhost:3000/login?username=${username}&password=${password}`, {
+      method: 'GET',
+      headers: {
+        'content-type': 'application/json'
+      }
+    })
+      .then(response => response.json())
+  }
+  const handleSubmit = async e => {
+    e.preventDefault()
+    const foundUser = await loginUser({
+      username,
+      password
+    })
+    setLoading(true)
+    setError(null)
+    if (setFoundUser(foundUser)) {
+      return <Home />
```

Comment:
Should not return a component to redirect to a page. Use history property from Router. https://reactrouter.com/web/api/history, maybe the replace func.

Reply:
i don't know it is true or false.
![image](https://user-images.githubusercontent.com/62316127/122877756-24587f00-d361-11eb-8433-e119fef1bb2d.png)

Reply:
should not use the history library. You can use the `history` property of the `Login` Component

Reply:
Because you have
```js
<Route exact path='/login' component={Login} />
```
then u can console.log history property in Login Component like this to see the history object. The history object should contain the `replace` function that you want.
```js
const Login = (props) => {
  console.log(props.history)
}
```

Reply:
![image](https://user-images.githubusercontent.com/62316127/123399622-73bdcb80-d56a-11eb-8b5a-27b747cbd359.png)